Sources/TuistKit/Commands/BuildCommand.swift

```diff
@@ -18,7 +36,7 @@ public struct BuildOptions: ParsableArguments {
     public var generate: Bool = false
 
     @Flag(
-        help: "When passed, it cleans the project before building it"
+        help: "[Deprecated] When passed, it cleans the project before building it"
     )
     public var clean: Bool = false
```

Comment:
We don't need to control this, nor `device` tightly, but these will be used that often that I would consider keeping them in our top-level CLI interface. In the case of `device`, I'd change it to `destination`, so it's more 1-1 with `xcodebuild`. Up to discussion, though

Reply:
As for `device`, if we want to deliver features such as [this](https://github.com/tuist/tuist/pull/6307) one, I do think it might be smart to keep control over it.

Reply:
That is a good point indeed. Maybe a compromised solution is to keep them around top-level (clean, destination-related, etc) to leverage other tuist features or configurations and not be as constrained as with `-workspace` or `-project` passthrough arguments, letting `xcodebuild` fail if they are also provided as passthrough argument?